src/common/markdown.ts

```diff
@@ -0,0 +1,18 @@
+import MarkdownIt from 'markdown-it';
+import hljs from 'highlight.js';
+
+const markdown: MarkdownIt = MarkdownIt({
```

Comment:
You can add `export` before the `const` instead of adding a new line at the bottom